_posts/2015/2015-11-24-essence-of-tasks.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "On the essence of the task"
+date: "2015-11-24 07:23:00"
+icon: tasks
+categories: tech-stories
+authors: ["Timo H."]
+---
+
+{% highlight text %}
+"Task = Stuff needs to be done, do it, and you are rid of it!"
+{% endhighlight %}
+
+The problem with tasks is however, that there are quite many of them.
+Infinite, to be exact.
+Some mathematician could say which infinite, but pretty much anyway.
+
+The point is that they cannot be all done, definitely not at once.
+Some of them are actually preventing that others of them can be done.
+Even the amount of the tasks for some limited subarea, for example for bringing the best e-commerce to small people who could not manage it alone, is in terms of human lifespan infinite.
```

Comment:
I dont fully understand what is being said here so please check with the author before accepting my changes

Even the number of the tasks for some limited subareas, for example, bringing the best e-commerce platform to small businesses who could not manage it alone, is in terms of human lifespan infinite.